src/tiles/grid/item-tile.ts

```diff
@@ -78,12 +80,24 @@ export class ItemTile extends LitElement {
    * Templates
    */
   private get creatorTemplate(): TemplateResult | typeof nothing {
-    if (!this.model?.creator) return nothing;
+    let displayedCreator = this.model?.creator;
+
+    // If we're filtering by creator initial and have multiple creators, we want
+    // to surface the first creator who matches the filter.
+    if (this.selectedCreatorFilter && this.model?.creators.length) {
+      const firstLetter = this.selectedCreatorFilter;
+      displayedCreator =
+        this.model.creators.find(creator =>
+          creator.toUpperCase().startsWith(firstLetter)
```

Comment:
Chat GPT recommends this to make it so that e.g. "E" still matches "émile" (which the search engine will match).

```suggestion
          creator..normalize("NFD").replace(/[\u0300-\u036f]/g, "").toUpperCase().startsWith(firstLetter)
```

Reply:
Good catch! I wonder if the `replace` step is actually necessary here... If that range of code points only includes combining characters that modify the character _before_ them, then the decomposition alone should still start with the bare letter... I think? (I'll subject this to some more testing 😄)

Reply:
Oh good thinking! ChatGPT agrees 😁 https://chat.openai.com/share/0d1e3d87-9515-40fb-a010-23f3d8bdb11d